### PR TITLE
Prepare statement

### PR DESCRIPTION
**Pull request by egedebruin**
This ensures that there is no SQL Injection possible anymore. The SQL queries are first prepared before adding the parameters in the queries, ensuring that there is only one query executed.

### CODE REVIEW

**Comment by Kingdorian** [comment]
I might have looked over them but  I don't see any tests to ensure SQL injection is not possible anymore. You might want to add those.

**Comment by egedebruin** [comment]
Good thinking, I will add thos tests.

File: backend/src/main/java/com/benine/backend/database/MySQLDatabase.java [hunk 1]
```diff
@@ -456,17 +452,24 @@ public void checkNewCameras(ArrayList<Camera> cameras, ArrayList<String> macs)
 
   @Override
   public void deleteCamera(int cameraID) {
-    Statement statement = null;
+    PreparedStatement statement = null;
```
Comment (jochem725) [inline backend/src/main/java/com/benine/backend/database/MySQLDatabase.java]: This method is becoming quite long now, maybe you can create a method to reduce the amount of code duplication.

**Comment by jochem725** [comment]
I've placed a small comment. If Dorian's comment is fixed, I'll agree with merging.

**Comment by egedebruin** [comment]
I have added the tests for the SQL Injection @Kingdorian

**Comment by bvanwijk** [comment]
I agree with merging.

**Comment by jochem725** [comment]
Looks good, I'll agree with merging.

**Comment by egedebruin** [comment]
First #197 needs to be merged in this.

**Comment by bvanwijk** [comment]
I think this can be merged now.

**Pull request merged**